src/transforms/textDecoration.js

```diff
@@ -14,30 +14,30 @@ module.exports = tokenStream => {
   let style
   let color
 
-  if (tokenStream.matches(NONE)) {
-    tokenStream.expectEmpty()
-    return {
-      $merge: {
-        textDecorationLine: defaultTextDecorationLine,
-        textDecorationStyle: defaultTextDecorationStyle,
-        textDecorationColor: defaultTextDecorationColor,
-      },
-    }
-  }
-
   let didParseFirst = false
   while (tokenStream.hasTokens()) {
     if (didParseFirst) tokenStream.expect(SPACE)
 
     if (line === undefined && tokenStream.matches(LINE)) {
-      line = tokenStream.lastValue
+      const lines = [tokenStream.lastValue]
 
       tokenStream.saveRewindPoint()
-      if (tokenStream.matches(SPACE) && tokenStream.matches(LINE)) {
-        line += ` ${tokenStream.lastValue}`
+      if (
+        lines[0] !== 'none' &&
+        tokenStream.matches(SPACE) &&
+        tokenStream.matches(LINE)
+      ) {
+        lines.push(tokenStream.lastValue)
+        if (lines[0] === lines[1]) {
+          throw new Error('Expected two different text decoration line styles')
```

Comment:
What about adding a test for this error?

Reply:
Huh. Not even sure why I added that. I should probably just let RN handle the invalid values

Reply:
Well, to me it seems a bit weird that some CSS parsing errors are thrown by `css-to-react-native` and others go all the way to RN. Of course in both cases the user will see the error.

Reply:
I think maybe the cut-off should be is if the parsing is ambiguous or incomplete we throw an error, otherwise let RN do it.

Reply:
ok, that make sense